minutes/2020-april-30.md

```diff
@@ -0,0 +1,80 @@
+# Meeting Minutes for April 30, 2020
+
+## Agenda
+
+### General
+
+Thomas has created an new private repository (https://github.com/oss-review-toolkit/ort-governance) with the ORT's charter which is basically LF's standard charter with some minor changes. Made the repository private until we sort out the basics of our project.
```

Comment:
"Made the repository private" duplicates what was already written before. Maybe: "The repository will be made public once we sort out the basics of our project."